models/dave_worlds/worlds/dave_new_bathy_example.world

```diff
@@ -0,0 +1,197 @@
+        <material>
+          <script>
+            <uri>file://Media/materials/scripts/waves.material</uri>
+            <name>UUVSimulator/SimpleWaves</name>
+          </script>
+        </material>
+      </visual>
+
+      <visual name="ocean_below">
+        <pose>0 0 0 3.14 0 0</pose>
+        <cast_shadows>false</cast_shadows>
+        <geometry>
+          <mesh>
+            <uri>file://Media/models/sea_surface_1000m_x_1000m.dae</uri>
+            <scale>4 4 4</scale>
+          </mesh>
+        </geometry>
+        <material>
+          <script>
+            <uri>file://Media/materials/scripts/water.material</uri>
+            <name>UUVSimulator/StaticWater</name>
+          </script>
+        </material>
+      </visual>
+      </link>
+    </model>
+
+    <!-- Heightmap -->
+    <model name="heightmap">
+      <static>true</static>
+      <link name="link">
+        <collision name="collision">
+          <geometry>
+            <heightmap>
+                <uri>file://Santorini_heightmap.tif</uri>
+                <!-- Scale for original size of terrain segment -->
+                <size>8090 6000 141</size>         
+                <pos>0 0 1400</pos>
+            </heightmap>
+          </geometry>
+        </collision>
+        <visual name="visual">
+          <geometry>
+            <heightmap>
+              <texture> <!-- Add sample texture for visualization -->
+                <diffuse>file://media/materials/textures/dirt_diffusespecular.png</diffuse>
+                <normal>file://media/materials/textures/flat_normal.png</normal>
+                <size>1</size>
+              </texture>
+              <uri>file://media/Santorini_heightmap.tif</uri>
+              <size>8090 6000 141</size>
+              <pos>0 0 1400</pos>
+             </heightmap>
+          </geometry>
+        </visual>
+      </link>
+    </model>
+
+    <plugin name="underwater_current_plugin" filename="libuuv_underwater_current_ros_plugin.so">
+      <namespace>hydrodynamics</namespace>
+      <constant_current>
+        <topic>current_velocity</topic>
+        <velocity>
+          <mean>0</mean>
+          <min>0</min>
+          <max>5</max>
+          <mu>0.0</mu>
+          <noiseAmp>0.0</noiseAmp>
+        </velocity>
+
+        <horizontal_angle>
+          <mean>0</mean>
+          <min>-3.141592653589793238</min>
+          <max>3.141592653589793238</max>
+          <mu>0.0</mu>
+          <noiseAmp>0.0</noiseAmp>
+        </horizontal_angle>
+
+        <vertical_angle>
+          <mean>0</mean>
+          <min>-3.141592653589793238</min>
+          <max>3.141592653589793238</max>
+          <mu>0.0</mu>
+          <noiseAmp>0.0</noiseAmp>
+        </vertical_angle>
+      </constant_current>
+    </plugin>
+
+    <plugin name="sc_interface" filename="libuuv_sc_ros_interface_plugin.so"/>
+
+    <gui fullscreen='0'>
+      <camera name='user_camera'>
+<!--    <pose frame=''>34.6042 32.2383 -20.6633 0 0.528384 1.41245</pose> -->
```

Comment:
Is this redundant?

Reply:
It's just copied over from the basic dave_ocean_waves.world file that I started from, so I didn't think much about it.  We do need to set the camera pose here, since that's not inherited from anything else.  Which part is redundant, though?  Would be good to clean this up.

Reply:
Sorry I meant the commented pose frame

Reply:
Done!